source/constructs/lib/admin/delete-resources/delete_resources.py

```diff
@@ -72,9 +72,9 @@ def on_delete(event):
 
 
 def refresh_account(event):
-    response = lambda_client.invoke(FunctionName=f'{solution_name}-RefreshAccount',
-                                    Payload='{"UpdateEvent":"RerefshAccount"}')
-    print(response)
+    response = lambda_client.invoke(FunctionName=f'{solution_name}-Controller',
+                                    Payload='{"Action":"RefreshAccount"}')
+    logger.info(response)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

You have a log statement that might use unsanitized input. Depending on the context, this could result in:

1. A log injection attack that breaks log integrity, forges log entries, or bypasses monitors that use the logs. To increase the security of your code, sanitize your inputs before logging them. [Learn more](https://cwe.mitre.org/data/definitions/117.html)

2. A sensitive information leak that exposes users&#39; credentials, private information, or identifying information to an attacker. To preserve privacy in your code, redact sensitive user information before logging it. [Learn more](https://cwe.mitre.org/data/definitions/532.html)